docs/topics/api/addons.rst

```diff
@@ -175,6 +175,9 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json string previews[].image_url: The URL (including a cachebusting query string) to the preview image.
     :>json int previews[].thumbnail_size[]: width, height dimensions of of the preview image thumbnail.
     :>json string previews[].thumbnail_url: The URL (including a cachebusting query string) to the preview image thumbnail.
+    :>json object|null promoted: Object holding promotion information about the add-on. Null if the add-on is not currently promoted.
+    :>json string promoted.category: The name of the promoted category for the add-on.
+    :>json array promoted.applications[]: Array of the application names for which the add-on is promoted.
```

Comment:
Link to the table that lists the application names (like `app` does above)

Reply:
... maybe this field should be called `apps` for parity with the `app` query field?  Not sure.

Reply:
A few places where `application` is referenced in the doc for a field it's called `app_name`. I do notice that it's never explicitly called `application`, so changing it to `apps` or `app_names` makes sense. The latter seem cumbersome, so I'll just use `apps`.